reports/pharmacy_first_report.Rmd

```diff
@@ -93,7 +121,7 @@ Links to the codelist for each analysis can be found beneath the relevant sectio
 
 ### Total population
 
-```{r, message=FALSE, warning=FALSE, fig.height=4, fig.width=4}
+```{r, message=FALSE, warning=FALSE}
```

Comment:
I think we should keep this? it makes sure all the panels are roughly the same size? or what was your reason for removing this?

Reply:
There was an issue with the titles and axes being cropped out